dashboard/src/features/shared/actions/create.js

```diff
@@ -7,6 +7,8 @@ export default function(type, options = {}) {
   const listPath = options.listPath || `/${type}s`
   const createPath = options.createPath || `${listPath}/create`
   const created = (param) => ({ type: `CREATED_${type.toUpperCase()}`, param })
+  const receivedKeys = (param) => ({ type: 'RECEIVED_MOCKHSM_ITEMS', param })
```

Comment:
I think this strategy will work and it is the smallest fix in terms of lines of code, but it may be an abuse of the reducer abstraction to use `RECEIVED_MOCKHSM_ITEMS` here. In other words, this may put the solution to the problem in the wrong part of the code. There's no other reason to make this file aware of the `RECEIVED_MOCKHSM_ITEMS` other than to fix a state inconsistency, but fixing state issues should happen in the reducers.

One other route to consider is expanding the reducer for the relevant item in the state tree ([this one?](https://github.com/chain/chain/blob/main/dashboard/src/features/mockhsm/reducers.js)) so that it handles both `RECEIVED_MOCKHSM_ITEMS` and `CREATED_MOCKHSM`.